buildimage.go

```diff
@@ -77,7 +84,7 @@ func main() {
 		{args: []string{"date"}},
 		{args: []string{"ls"}},
 		{args: []string{"pwd"}},
-		{args: []string{"go", "build"}, dir: "webboot"},
+		{args: []string{"go", "build"}, dir: filepath.Join(currentDir + "/webboot")},
```

Comment:
filepath.Join(currentDir, "webboot")

you don't need the / == Join does all that.